x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -163,11 +158,12 @@ export function ProcessTreeNode({
                 <span css={styles.darkText}>{args[0]}</span>&nbsp;
                 {args.slice(1).join(' ')}
                 {exitCode !== undefined && (
-                  <small data-test-subj="sessionView:processTreeNodeExitCode">
+                   <small data-test-subj="sessionView:processTreeNodeExitCode">
```

Comment:
more whitespace issues. Make sure your IDE is setup to format (w eslint) on save.